java-logger/src/main/filtered-resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<archetype-descriptor name="basic">
+  <requiredProperties>
+    <requiredProperty key="dhOrg">
+      <defaultValue>mydocker</defaultValue>
```

Comment:
not sure if we want a _better_ default for the dockerhub org of the _end-user_ ?